common/src/main/kotlin/action/Action.kt

```diff
@@ -21,20 +22,25 @@ open class Action(
         // Do nothing
     }
 
+    // this would work, right?
```

Comment:
I do believe so, maybe we should just make one large execute method rather than having a bunch of them, idk though.